obd_ontology/expert_knowledge_enhancer.py

```diff
@@ -252,12 +267,7 @@ def extend_knowledge_graph(self) -> None:
 
         if "dtc" in self.knowledge_file:
             dtc_knowledge = expert_knowledge_parser.parse_knowledge(self.knowledge_file)
-            dtc_uuid, dtc_facts = self.generate_dtc_facts(dtc_knowledge)
-            _, fault_cat_facts = self.generate_fault_cat_facts(dtc_uuid, dtc_knowledge)
-            fault_cond_uuid, fault_cond_facts = self.generate_fault_cond_facts(dtc_uuid, dtc_knowledge)
-            symptom_facts = self.generate_symptom_facts(fault_cond_uuid, dtc_knowledge)
-            diag_association_facts = self.generate_facts_to_connect_components_and_dtc(dtc_uuid, dtc_knowledge)
-            fact_list = dtc_facts + fault_cat_facts + fault_cond_facts + symptom_facts + diag_association_facts
+            fact_list = self.generate_all_dtc_related_facts(self, dtc_knowledge)
```

Comment:
`generate_all_dtc_related_facts` does not exist - you probably meant `generate_dtc_form_related_facts`? I'd suppose to call the function `generate_dtc_related_facts`.